src/Horse.Core.Param.pas

```diff
@@ -75,6 +75,7 @@   THorseCoreParam = class
     function ContainsValue(const AValue: string): Boolean;
     function ToArray: TArray<TPair<string, string>>;
     function TryGetValue(const AKey: string; var AValue: string): Boolean;
+    function AddOrSetValue(const AKey: string; const AValue: string): THorseCoreParam;
```

Comment:
These parameters come from the Request, that is, they shouldn't be changers and added values ​​on the outside.